apps/docs/src/getting-started/templates/blank-template.md

```diff
@@ -25,16 +25,18 @@ The blank template is pre-configured to connect to a public Shopware backend, so
 
 In order to connect it to your own store backend, you need to edit the `nuxt.config.ts` file and add a configuration details:
 
-```ts{4-7}
+```ts
 /* ... */
 export default defineNuxtConfig({
-  /* ... */
-  shopware: {
-    endpoint: "https://your-business.shopware.store",
-    accessToken: "access-token-from-settings",
-  },
   // or
```

Comment:
remove this comment as well it if you don't propose the way without runtimeConfig